core/store/dbutil/dbutil.go

```diff
@@ -3,21 +3,26 @@ package dbutil
 import (
 	"strings"
 
-	"github.com/jinzhu/gorm"
+	gormv1 "github.com/jinzhu/gorm"
+	"gorm.io/gorm"
 )
 
 func IsPostgresURL(url string) bool {
 	return strings.HasPrefix(strings.ToLower(url), "postgres")
 }
 
 // IsPostgres returns true if the underlying database is postgres.
-func IsPostgres(db *gorm.DB) bool {
+func IsPostgres(db *gormv1.DB) bool {
```

Comment:
This seems useless since we only support postgres now

Reply:
yeah its used all over the v1 migrations though, I'll leave it in until we deprecate that